tests/sentry/backup/test_sanitize.py

```diff
@@ -493,6 +494,7 @@ class SanitizationExhaustiveTests(BackupTestCase, IntegrationTestCase):
     were sanitized remain constant.
     """
 
+    @freeze_time("2024-01-01")
```

Comment:
@azaslavsky That's the reason this PR hasn't been merged yet, agree that this isn't ideal i've been trying to figure it out for a few days now.. The other test failures are failing for the same reason as this, this freeze time was added to confirm the issue i'm investigating.

The core issue is that i'm using the the `DefaultFieldsModel` class here: https://github.com/getsentry/sentry/pull/69816/files#diff-f0c227df676163e19911e53f7e40bb248f83c1e4d36d1248688c5b7ce0e9282aR8 This is causing there to be an `updated_at` change and causing the test failures. If I use freeze time it correctly isolates that change, but agree that this likely has additional side effects.

Any chance you know of how we could do this in a better way? Should I use a different base class? It's a little tricky since it's all happening behind the scenes.

Reply:
You need to add a new comparator like at https://github.com/getsentry/sentry/blob/acd19123ae11d8d77bad446869ea7b41818c94ee/src/sentry/backup/comparators.py#L808 for your model.

Reply:
Thanks so much!! that fixed it 🎉